docs/sdk-and-tools/notifier.md

```diff
@@ -244,7 +252,7 @@ There is a separate config section `WebSocketConnector` that has to be aligned w
 
 [comment]: # (mx-context-auto)
 
-#### HTTP Integration
+### HTTP Integration
```

Comment:
maybe add `{#websocket-http-integration}` since we already have a 'http-integration' and this one resolves to 'http-integration-1'. nice find with this custom solution 👍

Reply:
done